City-of-Helsinki/helsinki-paatokset
# PP-383: Use translated policymaker node titles for meetings

**To test**
- Checkout branch
- Run `make drush-cr`
- Run `make ahjo-migrations` if you don't have policymakers and recent meeting data yet
- Add an english translation for Kaupunginvaltuusto: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto (and swedish, if one doesn't exist yet) 
- The translated policymaker node title (not ahjo title) should be used on the following pages:
  - Different translations of the frontpage meeting calendar: https://helsinki-paatokset.docker.so/fi/etusivu
  - Different translations of the meeting calendar: https://helsinki-paatokset.docker.so/fi/kokouskalenteri
